modules/tinymce/src/core/main/ts/api/NotificationManager.ts

```diff
@@ -42,7 +42,7 @@ export interface NotificationApi {
 }
 
 interface NotificationManager {
-  open: (spec: NotificationSpec) => NotificationApi;
+  open: (spec: NotificationSpec, fireEvent?: boolean) => NotificationApi;
```

Comment:
We probably don't want this to be a public flag, it should really be internal only.

- [ ] Remove this from the public types